api/pom.xml

```diff
@@ -96,11 +95,12 @@
             <plugin>
                 <groupId>biz.aQute.bnd</groupId>
                 <artifactId>bnd-baseline-maven-plugin</artifactId>
-                <version>3.5.0</version>
+                <version>${bnd.version}</version>
                 <configuration>
                     <base>
-                        <version>2.0</version>
+                        <version>2.1</version>    
```

Comment:
```suggestion
                        <version>2.1</version>
```

Reply:
done

Reply:
The GitHub diff still shows the extra space at the end of line?